uxarray/grid/geometry.py

```diff
@@ -0,0 +1,199 @@
+import numpy as np
+
+from uxarray.constants import INT_DTYPE, INT_FILL_VALUE
+from uxarray.grid.connectivity import close_face_nodes
+
+
+def grid_to_polygons(grid, correct_antimeridian_polygons=True):
```

Comment:
As mentioned before, I don't think this should be a user API function

Reply:
The reason I chose to keep this as a user API function is that it returns an array of shapely polygons, which can then be manipulated by the user if they have any workflows that require the unstructured grid to be represented in Shapely.

Reply:
Consider a `Grid.to_shapely_polygons` conversion method instead of directly exposing this function to the user.